great_expectations/expectations/metrics/query_metrics/query_template_values.py

```diff
@@ -28,8 +28,8 @@ class QueryTemplateValues(QueryMetricProvider):
     @classmethod
     def get_query(cls, query, template_dict, selectable):
         template_dict_reformatted = {
-            k: sa.text(v).format(active_batch=selectable)
-            if isinstance(v, int)
+            k: str(v).format(active_batch=selectable)
+            if isinstance(v, (float, int))
```

Comment:
```suggestion
            if isinstance(v, float)
```
A an `int` is a subclass of `float`, so just checking for the `float` should be enough,.

Reply:
Tried this, but it doesn't work, unfortunately. It goes to the `else` branch, and gives the following error. 

```
E   AttributeError: 'int' object has no attribute 'format'
```

Reply:
That doesn't seem right based on this rule, maybe this was changed in a later version of python (or the rule is just wrong).
https://beta.ruff.rs/docs/rules/redundant-numeric-union/

You could do this...
```
import numbers

if isinstance(v, number.Number)
```


https://docs.python.org/3/library/numbers.html#numbers.Number

Reply:
oo sure. I can add this